README.md

```diff
@@ -53,56 +54,56 @@ scalar.db.storage=grpc
 # The type of the transaction manager
 scalar.db.transaction_manager=grpc
 ```
-- Setting up Scalar DB schema: As mentioned in [`Getting started with Scalar DB`](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb.md) storing and retrieving data in `Scalar DB` can be done using either the `storage` or the `transaction` mode.
-    - Our storage schema is  [`emoney-storage-schema.json`](./src/main/resources/emoney-storage-schema.json)
-    ```
-    {
-      "emoney_storage.account": {
-        "transaction": false,
-        "partition-key": [
-          "id"
-        ],
-        "clustering-key": [],
-        "columns": {
-          "id": "TEXT",
-          "balance": "INT"
+- Setting up Scalar DB schema: As mentioned in [Getting started with Scalar DB](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb.md) storing and retrieving data in `Scalar DB` can be done using either the `storage` or the `transaction` mode.
```

Comment:
```suggestion
Now you apply the database schema of the sample application as shown below. (It is stored in [emoney-storage-schema.json](./src/main/resources/emoney-storage-schema.json))
Note that this case uses non-transactional mode (storage mode) since `transaction` key is set to false.
```